testing/microlith-test/integration/prometheus_alert_overrides/prometheus_alert_overrides.bats

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env bats
+
+# Copyright 2021 Couchbase, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file  except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the  License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load "$HELPERS_ROOT/test-helpers.bash"
+
+verify_prerequisites BATS_SUPPORT_ROOT BATS_ASSERT_ROOT BATS_FILE_ROOT HELPERS_ROOT TEST_ROOT CMOS_IMAGE
+
+load "$BATS_SUPPORT_ROOT/load.bash"
+load "$BATS_ASSERT_ROOT/load.bash"
+load "$BATS_FILE_ROOT/load.bash"
+load "$HELPERS_ROOT/couchbase-helpers.bash"
+load "$HELPERS_ROOT/url-helpers.bash"
+
+setup() {
+    if [ "$TEST_INTEGRATION" == "true" ]; then
+        skip "Skipping integration tests"
+    fi
+    find "$BATS_TEST_DIRNAME/couchbase" "$BATS_TEST_DIRNAME/custom" "$BATS_TEST_DIRNAME/overrides" -name '*.orig' -delete
+}
+
+teardown() {
+    if [ "$SKIP_TEARDOWN" == "true" ]; then
+        echo "# Skipping teardown. Make sure to manually run the commands in teardown()." >&3
+        return
+    fi
+    run docker-compose --project-directory="${BATS_TEST_DIRNAME}" rm -v --force --stop
```

Comment:
Not quite sure what you mean by this - you mean putting in an `assert_success` or similar?

Reply:
Basically what I don't want is a load of error output from the teardown just because `docker-compose up` failed much earlier - we should aim to just get the output from the failing command.
I think this might be ok unless `run` dumps all the output on error?

Reply:
I'm not sure on this - we could stick in a `>/dev/null 2>&1`, but if teardown fails I'd like to know about it so I'm not confused when my next test run hits dirty state - and I'm not sure if there's a good way of consuming output _only_ if something failed previously.